app/src/main/java/org/fossasia/openevent/general/search/SearchTimeActivity.kt

```diff
@@ -13,6 +13,11 @@ import java.text.SimpleDateFormat
 import java.util.*
 
 const val ANYTIME = "Anytime"
+const val TODAY  = "Today"
+const val TOMORROW = "Tomorrow"
+const val WEEKEND = "This Weekend"
+const val MONTH = "In the next month"
```

Comment:
Give meaningful names

Reply:
I am talking about the variables

Reply:
I'm bad with naming variable pls suggest one 😜